<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnLogo">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="zcontrol.qrc">
          <normaloff>:/pict/logo</normaloff>:/pict/logo</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>96</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnHingeLeft">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>96</height>
         </size>
        </property>
        <property name="text">
         <string>Hinge Left</string>
        </property>
        <property name="icon">
         <iconset resource="zcontrol.qrc">
          <normaloff>:/pict/hinge</normaloff>:/pict/hinge</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnHingeRight">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>96</height>
         </size>
        </property>
        <property name="text">
         <string>Hinge Right</string>
        </property>
        <property name="icon">
         <iconset resource="zcontrol.qrc">
          <normaloff>:/pict/hinge</normaloff>:/pict/hinge</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnDebug">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>Debug/Logging</string>
        </property>
        <property name="icon">
         <iconset resource="zcontrol.qrc">
          <normaloff>:/pict/bug_green</normaloff>:/pict/bug_green</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnTelegram">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>Telegram</string>
        </property>
        <property name="icon">
         <iconset resource="zcontrol.qrc">
          <normaloff>:/pict/telegram</normaloff>:/pict/telegram</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="presentation">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Verdana Pro</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CAM do Brasil Ltda</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hinge">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,1">
          <item>
           <widget class="QLabel" name="lblLive">
            <property name="text">
             <string>Live:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leLiveCounter">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Step Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leStepNumber">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leStepDescription">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="btnPrev">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>Previous Step</string>
            </property>
            <property name="icon">
             <iconset resource="zcontrol.qrc">
              <normaloff>:/pict/prev</normaloff>:/pict/prev</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnNext">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>Next Step</string>
            </property>
            <property name="icon">
             <iconset resource="zcontrol.qrc">
              <normaloff>:/pict/next</normaloff>:/pict/next</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnReset">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>Reset Step</string>
            </property>
            <property name="icon">
             <iconset resource="zcontrol.qrc">
              <normaloff>:/pict/reset</normaloff>:/pict/reset</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnStepByStep">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>Step By Step</string>
            </property>
            <property name="icon">
             <iconset resource="zcontrol.qrc">
              <normaloff>:/pict/ladder</normaloff>:/pict/ladder</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="debug">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Verdana Pro</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Debug</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="toPc"/>
        </item>
        <item>
         <widget class="QTableView" name="tblLocal"/>
        </item>
        <item>
         <widget class="QTableView" name="tblRemote"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="telegram">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Verdana Pro</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Telegram</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="logTelegram">
          <property name="source">
           <url>
            <string>http://www.liprandi.com/projects/betim/zServicePHP.php</string>
           </url>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionStart_Ethernet_IP"/>
    <addaction name="actionStop_Ethernet_IP"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Ethernet_IP">
   <property name="text">
    <string>Start Ethernet/IP</string>
   </property>
  </action>
  <action name="actionStop_Ethernet_IP">
   <property name="text">
    <string>Stop Ethernet/IP</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="zcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
